Cryptography/Cryptography/Cryptography/SymmetricEncryptor.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System.IO;
+using System.Security.Cryptography;
+
+namespace Cryptography
+{
+    /// <summary>
+    /// Encrypts and Decrypts objects symmetrically.
+    /// </summary>
+    public static class SymmetricEncryptor
+    {
+        /// <summary>
+        /// Takes a string and encrypts it using symmetric encryption.
+        /// </summary>
+        /// <param name="plainText">The string to encrypt.</param>
+        /// <param name="key">The key for the encrypion.</param>
+        /// <param name="iv">The iv for the encryption.</param>
+        /// <returns></returns>
```

Comment:
It would be good to include a description of what the method returns as well for your callers.